recipes/rasa-sdk/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "rasa-sdk" %}
+{% set version = "3.0.0rc3" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 162e72dc3c141d9f5220f3de921be6c902e03f209980a42cdb2cb1ff21f534d6
+  
+
+build:
+  skip: True  # [osx or win]
+  number: {{ build_num }}
+
+outputs:
+  - name: {{ name|lower }}    
+    build:
+      skip: True  # [osx or win]
+      noarch: python
+    script: build_base.sh
+    requirements:
+      {% set pyver = ">=3.7,<3.10" %}
+      host:
+        - pip
+        - python {{ pyver }}
+        - poetry-core >=1.0.0
```

Comment:
Not sure why this is not working. I would ask in the general chat again.

Reply:
Thanks. Will do.